### ISSUE TITLE
Syntax error but no clue

### ISSUE THREAD

**Issue by f4lco** [Kotlin/kotlin-jupyter#50]
I'm using kotlin-jupyter-kernel 0.7.40.145 and a cell with the following code (and syntactic error)  produces no output:

```kotlin
data class Person(val name: String, val age: Int)
val example = Person("Testperson"; age = 4)
println(example)
```
As evident in this screen shot:
![person](https://user-images.githubusercontent.com/29141485/77435889-1ffe8880-6de3-11ea-8f77-a0285026e8a5.png)

Whenever I make a different kind of error, such as leaving out a parameter (as in `Person("foo")`), I get a helpful message in red below the cell. There should be a similar error message when mistaking `;` for `,`.

**Comment by ileasile** [comment]
Hello, f4lco! Please check it for version **0.7.41.254.dev1** (may be installed from dev channels, see Readme for instructions). Should work now.
![image](https://user-images.githubusercontent.com/20319593/79812597-15072b80-8382-11ea-91f7-6ecce399b318.png)

**Comment by f4lco** [comment]
Hello @ileasile this issue appears fixed in [**0.7.41.254.dev1**](https://test.pypi.org/project/kotlin-jupyter-kernel/0.7.41.254.dev1/) as advertised (minus the fancy tooltip - maybe it requires additional dependencies or configuration?).

My first attempt - running `pip install -i https://test.pypi.org/simple/ kotlin-jupyter-kernel`, because it is labeled as "dev" in the README - installed [**0.7.41.190**](https://test.pypi.org/project/kotlin-jupyter-kernel/0.7.41.190/). In this version the bug is also fixed, but in a slightly different manner:

<img width="483" alt="kotlin-jupyter-kernel-0 7 41 190" src="https://user-images.githubusercontent.com/29141485/79858932-7a820900-83d0-11ea-8d5c-07b281640719.png">

I thought it might be of interest to you, in case this is unexpected.

Both releases clear the bar for me and provide some indication what's wrong with the code (which line to look at and what to fix). Thank you very much, closing this now.

**Comment by f4lco** [comment]
I almost forgot the praise I was going to give. I used this kernel when recently trying to befriend some Java devs with Kotlin. Using the presentation mode of a Jupyter notebook with some prepared examples in addition to  livecoding really proved worthwhile and felt superior to using only a REPL. You guys made it possible, I really appreciate it. Thanks!
